spellbook/models/safe/ethereum/safe_ethereum_schema.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+models:
+  - name: safe_ethereum_safes
+    meta:
+      blockchain: ethereum
+      project: safe
+      contibutors: scherbovich, tschubotz
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'ethereum']
+    description: "Safe addresses"
+    columns:
+      - name: address
```

Comment:
minor feedback, but good to know for now & future spell builds:
you can enable "reuse" of columns downstream in the same file, so if two models have the same column, you can reuse it without defining twice. an example in this file is 'address' column in both. could look like:
      - &address
        name: address
        description:  "Safe contract address"

Reply:
comment continued on line 37